### ISSUE TITLE
Dead Project?

### ISSUE THREAD

**Issue by MrMEEE** [MS71/IndegoRover#2]
Hi

Are you still working on this / did you ever get it to work?

Have just gotten an old Indego, that I plan to transform...

**Comment by MS71** [comment]
Yes,will provide details this evening.Where do you come from?Am 01.08.2024 10:27 schrieb Martin Juhl ***@***.***>:
Hi
Are you still working on this / did you ever get it to work?
Have just gotten an old Indego, that I plan to transform...

**Comment by MrMEEE** [comment]
Cool... I'm from Denmark

**Comment by larsenglund** [comment]
I also have a couple of dead Indegos and would be interested in reviving them :) I'm in Sweden.

**Comment by MS71** [comment]
Hi,
yes i am still working on that.
Meantime, i am running several conventional Idego's  in my garden.
They are waiting for a new ROS PCB.

Just now i am preparing a prototype implementing powerswitch, powersupply, charger, motor driver.
After that, i will try connecting all other sensors step by step.
There is nothing finished or ready to use for now.

My wish is, having a first PCB this year.

If you have experience with electronics, ESP32, Linux/RPI and embedded software, then we can find a way 
sharing the work. For that we need some direct communication channel. WhatsApp, Discord or similar.
Please send a PN.

Maik

**Comment by MrMEEE** [comment]
I'm pretty good with Electronics and have been working with Linux and Opensource for 15 years.. So I guess I can chip in on those..

**Comment by MS71** [comment]
Hi Lars && MrMEEE,
would be great if we could get some direct contact.
Please send me a PM with your EMail address.

Currently, i am working on something like a prototype.
Goal is, interfacing the mower IO. 

Please contact me if you are still interested.

Maik

**Comment by MrMEEE** [comment]
Can't seem to find a way to send you a PM?
